src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/CrossCaseTypeAliasFuzzySearchOnText.scala

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+
+object CrossCaseTypeAliasFuzzySearchOnText extends PerformanceTestsConfig {
```

Comment:
Can we move all ES tests to another branch? Should not be part of this PR for create case data?

Reply:
Yes Moved Elastic Search, Cross Case type stuff in separate branch.